src/protocol.h

```diff
@@ -297,6 +299,9 @@ enum ServiceFlags : uint64_t {
  * Thus, generally, avoid calling with peerServices == NODE_NONE.
  */
 static ServiceFlags GetDesirableServiceFlags(ServiceFlags services) {
+    if (initialBlockDownloadCompleted) {
+        return ServiceFlags(NODE_NETWORK_LIMITED | NODE_WITNESS);
```

Comment:
This would result in us *only* connecting to NODE_NETWORK_LIMITED after IBD - I think this should be:
```
if (services & NODE_NETWORK_LIMITED && initialBlockDownloadCompleted) return ServiceFlags(NODE_NETWORK_LIMITED | NODE_WITNESS);
return ServiceFlags(NODE_NETWORK | NODE_WITNESS);
```